src/uwtools/cli.py

```diff
@@ -191,7 +191,7 @@ def _dispatch_config_compare(args: Args) -> bool:
 
     :param args: Parsed command-line args.
     """
-    return uwtools.config.tools.compare_configs(
+    return uwtools.api.config.compare(
```

Comment:
Here are below, the CLI now accesses `uwtools` library functionality via the API.